org.eclipse.jdt.core/dom/org/eclipse/jdt/core/dom/GuardedPattern.java

```diff
@@ -168,6 +166,7 @@ int treeSize() {
 
 	 * @return a list of property descriptors (element type:
 	 * {@link StructuralPropertyDescriptor})
+	 * @since 3.37
```

Comment:
Since this is now a standard feature in Java 21, the @noreference has been removed from the doc. Therefore, @since added here to signify the version introduction.